docs/updates/2019-03.md

```diff
@@ -9,18 +9,22 @@ Because of a security concern and to follow best-practices, the Qlik Associative
 user and group from `nobody:nobody` to `1910:1910`. You can read more about using an unique UID/GID here:
 [https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user).
 
-If you are on Mac or Linux and using Docker volume mounts, you may need to change the owner of the volume files. You can
-do that with the following command:
+**If you are on Mac or Linux and using Docker volume mounts, you may need to change the owner of the volume files. You can
+do that with the following command:**
 
 ```bash
 chown -R 1910:1910 /YOUR-MOUNT
 ```
 
+The change is effective from Qlik Associate Engine version 12.345.0.
```

Comment:
Is the change included in version `12.345.0`? Thought it was `12.350.0`?

Reply:
It should be 350!